### PR TITLE
chore: add SHA headers to some files in `Init`

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
A lot of the theorems in these files are not aligned. I'd prefer to review them via looking at the diff after #5945

**Comment by Parcly-Taxel** [comment]
> A lot of the theorems in these files are not aligned. I'd prefer to review them via looking at the diff after #5945

Done via force-push. Each commit with message "restore" has one of the mathport diffs.

File: Mathlib/Init/Data/List/Default.lean [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/-
```
Comment (semorrison) [inline Mathlib/Init/Data/List/Default.lean:1]: What is the purpose of having this file?

Comment (Parcly-Taxel) [reply Mathlib/Init/Data/List/Default.lean:1]: To remove an entry in the port dashboard.

**Pull request closed without merging**